packages/util-node-http-modules/src/util-node-http-modules.js

```diff
@@ -1,12 +1,19 @@
 export async function getNodeHttpModules() {
-  let nodeHttpsTransport
-  try {
-    nodeHttpsTransport = await import("https").catch(e => null)
-  } catch (e) {}
-  let nodeHttpTransport
-  try {
-    nodeHttpTransport = await import("http").catch(e => null)
-  } catch (e) {}
+  let nodeHttpsTransport = null
+  let nodeHttpTransport = null
+
+  if (
+    typeof process !== "undefined" &&
+    Object.prototype.toString.call(process) === "[object process]"
+  ) {
```

Comment:
The check whether we are running in node is more of a precaution and "good practice".  In actuality, if the environment did not support `http`/`https` modules it would be caught in the exception handling below